ocaml/xapi/helpers.ml

```diff
@@ -1153,3 +1153,16 @@ let force_loopback_vbd ~__context =
 	let other_config = Db.Pool.get_other_config ~__context ~self:pool in
 	List.mem_assoc "force_loopback_vbd" other_config
 
+let assert_virt_hw_support ~__context ~vm_virt_hw_vn ~host_to =
```

Comment:
Indentation/formatting is a bit mixed up in this function.
Also I think it would be better in xapi_vm_helpers.ml